egasub/cli.py

```diff
@@ -49,17 +49,24 @@ def report(ctx, source):
 
 
 @main.command()
+@click.argument('credentials',nargs=-1)
```

Comment:
`@click.option` should be a better choice here. Argument is position based, option is parameter-value based which is more flexible, allowing users to selective choose which option(s) to use (which not).